src/persist-client/src/write.rs

```diff
@@ -153,6 +154,7 @@ where
     pub async fn append<SB, KB, VB, TB, DB, I>(
         &mut self,
         updates: I,
+        lower: Antichain<T>,
         new_upper: Antichain<T>,
```

Comment:
genuine question, i'm unsure myself: should we call this upper instead? then append gets lower+upper and CaA gets expected_upper+new_upper?

Reply:
It's though: my thinking was that we could even do `lower, upper`, because they describe the batch of updates this is appending. But ... idk.

The good book (aka `formalism.md`) doesn't have a `lower` for `Append`, only `new_frontier`, so that's also not helping.

Reply:
> It's though: my thinking was that we could even do lower, upper, because they describe the batch of updates this is appending. But ... idk.

I'm not sure I follow. Is this saying the same thing I'm saying? :)

Reply:
Ah, yes!